src/status_im2/contexts/activity_center/notification/community_request/view.cljs

```diff
@@ -0,0 +1,35 @@
+(ns status-im2.contexts.activity-center.notification.community-request.view
+  (:require [quo2.core :as quo]
+            [status-im2.contexts.activity-center.notification.common.style :as common-style]
+            [status-im2.contexts.activity-center.notification.common.view :as common]
+            [utils.datetime :as datetime]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn- get-header-and-body-text
+  [membership-status]
+  (case membership-status
+    1 {:header-text (i18n/label :t/community-request-pending)
```

Comment:
Better not use magical constants like these @smohamedjavid, especially the ones referring to state/status. If the constants don't exist yet, we should create them based on what status-go uses.

Reply:
My bad, Apologies. Updated the constants. 👍